modules/apps/commerce/commerce-frontend-impl/src/main/java/com/liferay/commerce/frontend/internal/cart/CommerceCartResource.java

```diff
@@ -304,10 +327,327 @@ private Response _getResponse(Object object) {
 	@Reference
 	private CommerceOrderItemService _commerceOrderItemService;
 
+	@Reference
+	private CommerceOrderPriceCalculation _commerceOrderPriceCalculation;
+
 	@Reference
 	private CommerceOrderService _commerceOrderService;
 
+	@Reference
+	private CommerceOrderValidatorRegistry _commerceOrderValidatorRegistry;
+
+	@Reference
+	private CommercePriceFormatter _commercePriceFormatter;
+
+	@Reference
+	private CPInstanceHelper _cpInstanceHelper;
+
+	@Reference
+	private Language _language;
+
 	@Reference
 	private Portal _portal;
 
+	@Reference
+	private ProductHelper _productHelper;
+
+	private class CommerceCartResourceUtil {
+
+		public Cart getCart(
```

Comment:
just move the methods from CommerceCartResourceUtil to the outer class. No need to have an inner class here.